ui: remove node summary flickering [backport 21.1]

vladlos opened the pull request:
before: to avoid display wrong values on metrics page - summary
initial load we was hiding this block completly, but this leads to
flickering when new data was requested

after: change condition for hide smmary only when we have no
data to display (initial load). after that we display values as long as
we do not have new to display. do not hide summary block while
request in flight and data become "not valid".

Resolves: #61729

Release note(ui): fix flickering of summary block on metrics page

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/67365)
<!-- Reviewable:end -->

nathanstilwell reviewed (approved):
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/67365)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @koorosh and @Santamaura)



<!-- Sent from Reviewable.io -->

Santamaura reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/67365#-:-Me5hmTz4ZNQ2C6Hi9Nv:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @koorosh)



<!-- Sent from Reviewable.io -->

koorosh reviewed (approved):
> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/67365)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @elkmaster)



<!-- Sent from Reviewable.io -->

vladlos committed:
ui: remove node summary flickering

before: to avoid display wrong values on metrics page - summary
initial load we was hiding this block completly, but this leads to
flickering when new data was requested

after: change condition for hide smmary only when we have no
data to display (initial load). after that we display values as long as
we do not have new to display. do not hide summary block while
request in flight and data become "not valid".

Resolves: cockroachdb#61729

Release note(ui): fix flickering of summary block on metrics page
